runtimes/nodejs/actions/bucket-cors-delete.js

```diff
@@ -46,14 +52,32 @@ async function main(args) {
 
 
 function getParamsCOS(args, COS) {
+  var bxCredsApiKey = ""
+  var bxCredsResourceInstanceId = ""
+
+  if (args.__bx_creds && args.__bx_creds['cloud-object-storage']) {
+    if (args.__bx_creds['cloud-object-storage'].apiKey) {
+      bxCredsApiKey = args.__bx_creds['cloud-object-storage'].apikey
+    }
+    if (args.__bx_creds['cloud-object-storage'].resource_instance_id) {
+      bxCredsResourceInstanceId = args.__bx_creds['cloud-object-storage'].resource_instance_id
+    }
+  }
+
   const { bucket } = args;
   const endpoint = args.endpoint || 's3-api.us-geo.objectstorage.softlayer.net';
   const ibmAuthEndpoint = args.ibmAuthEndpoint || 'https://iam.cloud.ibm.com/identity/token';
-  const apiKeyId = args.apikey || args.apiKeyId || args.__bx_creds['cloud-object-storage'].apikey;
-  const serviceInstanceId = args.resource_instance_id || args.serviceInstanceId || args.__bx_creds['cloud-object-storage'].resource_instance_id;
+  const apiKeyId = args.apikey || args.apiKeyId || bxCredsApiKey || process.env.__OW_IAM_NAMESPACE_API_KEY;
+  const serviceInstanceId = args.resource_instance_id || args.serviceInstanceId || bxCredsResourceInstanceId;
```

Comment:
hi will - due to the variety of places the actual parameter values come from, i'm thinking that it will be helpful debug information to know not the value, from from where the parameter value was obtained.  for example, log to console that the apikey came from args.apikey or from the bxcreds or from namespace api key env var.  the values passed as parameters will be in the activation record, but values obtained from the env will be inferred.  now that i look at the above list of parameters, we might actually have enough information in the activation record already to infer which value was used.  do you happen to have an activation records (for any representative action) using the above code that (1) use a bxcreds api key and (2) use the __OW_IAM_NAMESPACE_API_KEY env var?